src/ontology/mondo-ingest-odk.yaml

```diff
@@ -25,18 +25,20 @@ documentation:
   documentation_system: mkdocs
 components:
   products:
+    - filename: doid.owl
+      source: http://purl.obolibrary.org/obo/doid.owl
+    - filename: gard.owl
+      source: https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl
+    - filename: icd10cm.owl
+      source: https://data.bioontology.org/ontologies/ICD10CM/submissions/21/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
```

Comment:
```suggestion
      source: https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
```

Reply:
Ahh, I see. Too bad we have to update this periodically. I'll commit this ICD10CM download URL change to `main` directly.